docs/reference/setup/install/docker.asciidoc

```diff
@@ -66,7 +66,7 @@ Depending on your platform:
 
 * Linux
 +
-The `vm_map_max_count` setting should be set permanently in /etc/sysctl.conf:
+The `vm_max_map_count` setting should be set permanently in /etc/sysctl.conf:
```

Comment:
I think this should be changed to `vm.max_map_count` as well. Can you push that change?